manifests/config/cm_opr-paas.yaml

```diff
@@ -34,6 +34,19 @@ data:
             - monitoring-edit
           argo-service-applicationset-controller:
             - monitoring-edit
+          serviceaccounts:
+            - argo-service-applicationset-controller
+            - argo-service-argocd-application-controller
+        extra_fields:
+          gitUrl:
+            required: true
+            validation: "https://github.com/[a-z0-9-/]+/[a-z0-9-/]+"
+          gitRevision:
+            default: "main"
+            validation: ".*"
+          gitPath:
+            defaullt: "."
```

Comment:
This typo will be caught once we move the paasConfig to a CRD. I'm however curious why this passes the current tests?